Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.19-alpine
+
+WORKDIR /app
+
+RUN mkdir -p github.com/cryptag/leapchat/miniware
+
+WORKDIR /app/github.com/cryptag/leapchat
+
+COPY miniware/* miniware/
+COPY *.go .
+COPY go.* .
+COPY LICENSE.md .
+
+RUN go build -o /leapchat
+
+CMD ["/leapchat"]
```

Comment:
@elimisteve I had this in my own branch. Force-pushed to resolve the conflict with your last commit.

Since you're making changes, I'm going to stop working for now. :)

Reply:
Hey Jim, didn't mean to colonize your branch! You finished creating `upgrade_frontend` fast and I should have let you come back to this branch unmolested.

These changes look great.